core/src/verify.rs

```diff
@@ -740,6 +746,7 @@ mod test {
             timestamp: 4,
             transactions_hash: agenda_transactions_hash,
             height: csv.header.height + 1,
+            previous_block_hash: csv.header.to_hash256(),
```

Comment:
Why is some case `csv.header.to_hash256()` and other cases `csv.header.previous_hash`?
Isn't `csv.header.to_hash256()` correct?

Reply:
Yes, you are correct. I made the subsequent changes on the following commit.